### DISCUSSION TITLE
Can't get Rofi show asking which application to use.

### DISCUSSION THREAD

**Discussion by Samueru-sama** [Anomalocaridid/handlr-regex#31] [Q&A] [answered]
I've been trying to set this up in Arch linux, I installed the Aur package and edited handlr.toml to enable_selector = true. 

I then added several applications (freetube, brave and ferdium), which resulted in this entry being created on mimeapps.list: 

x-scheme-handler/https=freetube.desktop;brave-browser.desktop;ferdium.desktop;

However now when I click on a link only ferdium opens, which is the first app on the line, I don't get the rofi pop up p asking which app to use, if I swap the order of the entries and say I put brave frist, then all links open in brave instead of freetube, still without the rofi window asking for which one to use. 

I'm using Arch linux with i3wm and rofi.

**Comment by Anomalocaridid** [comment]
~~Moved to issue #32.~~

Edit: Actually, it's probably better to discuss this here on second thought. My bad.

**Comment by Anomalocaridid** [comment, marked as answer]
@Samueru-sama does this happen when you open a url with handlr open in a terminal or just when you click it? If not, do you have [xdg-utils-handlr](https://aur.archlinux.org/packages/xdg-utils-handlr) installed?

**Reply by Samueru-sama** [reply]
Well this is embarrassing, yeah I didn't have xdg-utils-handlr installed, I thought this fork replaced it lol. Now it works like it should, thanks a million! 

~~By the way is it possible to set default apps based on the url type so that it opens those apps by default instead of having to select them?~~

Edit: Nvm, I've just added the example regex handler that does that, finally I've found this, I was baffled when reading for solutions that people only suggested using an extension on the web browser to redirect to freetube.

**Reply by Anomalocaridid** [reply]
Glad I could help!
